msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -120,7 +124,8 @@ void acquireToken(@NonNull final Activity activity,
                       @Nullable final String loginHint,
                       @NonNull final UiBehavior uiBehavior,
                       @Nullable final List<Pair<String, String>> extraQueryParameters,
-                      @NonNull final AuthenticationCallback callback);
+                      @NonNull final AuthenticationCallback callback
+    );
```

Comment:
nit:Why do we need to have these diffs i.e. why moving the bracket to the next lines?

Reply:
The Google Style Guide (which we follow for the Android code) doesn't _explicitly_ address this for interfaces, but the code style is generally to line-break/left-justify the closing braces. In this case since there are no braces to scope-on, I use the parens and apply the same style.

> [Line break before the closing brace.](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style)

Reply:
PS: Acknowledge this creates noise in the PR